satpy/readers/mviri_l1b_fiduceo_nc.py

```diff
@@ -565,6 +565,9 @@ def __init__(self, filename, filename_info, filetype_info,
         self.projection_longitude = float(filename_info['projection_longitude'])
         self.calib_coefs = self._get_calib_coefs()
 
+        self._get_angles = functools.cache(self._get_angles_uncached)
+        self._get_acq_time = functools.cache(self._get_acq_time_uncached)
```

Comment:
As you probably know, you have to use `lru_cache` here as `cache` isn't available until Python 3.9.

Reply:
Thanks for the hint, that was a testing leftover